src/autosuggest/internal.tsx

```diff
@@ -179,7 +180,7 @@ const InternalAutosuggest = React.forwardRef((props: InternalAutosuggestProps, r
   const highlightedOptionIdSource = useUniqueId();
   const highlightedOptionId = autosuggestItemsState.highlightedOption ? highlightedOptionIdSource : undefined;
 
-  const isEmpty = !value && !autosuggestItemsState.items.length;
```

Comment:
I understood why you added this change after playing with the new playground page (which is great 👍).

However, I think it would be great to have the interaction between hideEnteredTextLabel, empty text, and options be explicitly covered with unit tests.

I recommend designing 7 test cases that represent all possible combination of the states and where each test asserts the presence of the dropdown, the presence of the "Use: ..." option, the presence of Empty text, and the presence of options. Like:

```
it('hideEnteredTextLabel=true, emptyText=true, options=false', () => {
  // render
  // assert
})
```

What do you think?

Reply:
Sounds like a good idea! I added some unit tests in [66e066b](https://github.com/cloudscape-design/components/pull/2526/commits/66e066b57fc71f21c98863ffb297ee38dbd5004b), let me know what you think